playbooks/common/openshift-cluster/redeploy-certificates/redeploy-certificates.yml

```diff
@@ -0,0 +1,10 @@
+---
+- include: control-plane.yml
+
```

Comment:
We probably want to have a master/control-plane restart (option as well). The thought here, is to provide a way to just restart the control plane if there is an issue (with the general playbook).